### ISSUE TITLE
Karma tests not running on /test/unit/specs/mount/**/*.js

### ISSUE THREAD

**Issue by eddyerburgh** [eddyerburgh/avoriaz#10]
Karma tests aren't being run on files in the mount directory of the avoriaz project.

I have tried editing the karma.conf to catch the files with a glob pattern ('../specs/**/*.{vue,js}'), but it was throwing errors.

I don't have much experience with Karma, so if anyone knows how to fix please open a PR!

**Comment by matrunchyk** [comment]
Hmm it works for me. Do you have any error messages/screenshots/configs?

These are my configs.
[karma.conf.js](https://gist.github.com/matrunchyk/a79accea739f586074e005e0542200ea)
[index.js](https://gist.github.com/matrunchyk/31f910a35a2d149305af72c7aa113be2)

**Comment by eddyerburgh** [comment]
Hi @matrunchyk.

To be clear - the karma tests aren't running in the avoriaz project. 

I moved all the mount specs into their own directory - specs/mount . Karma isn't running any of the tests in the mount directory.

I tried using a global pattern to match the subdirectories in specs, but could not get it to work.

If you want to test, download avoriaz, ```yarn install``` and run ```yarn run test:karma```.

I'd like to fix this, but don't have too much time in the coming weeks to fiddle with the karma config.

I think it's probably an easy fix if

Thanks
